src/components/Image/Image.stories.mdx

```diff
@@ -98,22 +98,22 @@ export const imageBlockStyles = {
       additionalImageStyles: undefined,
       additionalWrapperStyles: undefined,
       alt: "Alt text",
+      aspectRatio: "ImageRatios.TwoByOne",
+      caption: "Image caption",
       className: undefined,
       component: undefined,
-      imageAspectRatio: "ImageRatios.TwoByOne",
-      imageCaption: "Image caption",
-      imageCredit: "Image credit",
-      imageSize: "ImageSizes.Medium",
+      credit: "Image credit",
       imageType: "ImageTypes.Default",
```

Comment:
Wondering why we removed the `image` prefix from all but type?

Reply:
Ah, because this `imageType` is related to the Chakra "variant" for its style. We were thinking of using `type` but the problem is that `type` is an actual HTML attribute and that will be confusing. For example, the HTML button element can take a `type` attribute and it can be set to "submit", "button", or "reset" `<button type="submit">Submit</button>`. To be consistent, the prop to change the variant is `buttonType`, while there also is a separate `type` prop for "submit", "button", and "reset" `<Button buttonType={ButtonTypes.Secondary} type="submit">Submit</Button>`.